app/js/utils/sorters.js

```diff
@@ -0,0 +1,20 @@
+export function weightedScore(user) {
+  const upvotes = user.upCount + 4;
+  const downvotes = user.downCount + 4;
+  return upvotes / (upvotes + downvotes);
```

Comment:
out of curiosity, why the `+ 4`

Reply:
My way of implementing a Bayesian weighted average (https://en.wikipedia.org/wiki/Bayesian_average)

Basically, it's the predicted amount of likes and dislikes, so it balances the fact that a user with 1 like and 0 dislikes shouldn't be higher than someone with 10 likes and 1 dislike.

Tell me if it's not clear and also if maybe I did it wrong. My Bayes classes were a long time ago

Reply:
Humm it is not really the formula, also I understand what you are doing.
To make a mapping: C == 4, m == 1 and sum == upCount or downCount == n
Correct?

Reply:
Yeah something like that. 
I more specifically followed this: http://julesjacobs.github.io/2015/08/17/bayesian-scoring-of-ratings.html